### PR TITLE
Fix Windows build: filesystem

### PR DESCRIPTION
**Pull request by chapulina**
I was too worried about test failures on #2765 that I didn't notice one of the builds was completely broken, sorry about that. This is an attempt at a fix.

### CODE REVIEW

File: gazebo/common/CMakeLists.txt [hunk 1]
```diff
@@ -264,9 +264,6 @@ endif()
 if (UNIX AND NOT APPLE)
   # rt is used for clock_gettime, which is not available on apple or windows
   target_link_libraries(gazebo_common PRIVATE rt)
```
Comment (ahcorde) [inline gazebo/common/CMakeLists.txt:266]:
```suggestion
  target_link_libraries(gazebo_common PRIVATE rt stdc++fs)
```

**Review by ahcorde — APPROVED** [review]
LGTM, just a minor comment. can we use only one `target_link_libraries` ?

**Comment by ahcorde** [comment]
I can't see the build results. The server is not responding.

**Comment by scpeters** [comment]
windows is fixed: [Build Status](https://build.osrfoundation.org/job/gazebo-ci-pr_any-windows7-amd64/3002/) https://build.osrfoundation.org/job/gazebo-ci-pr_any-windows7-amd64/3002/

**Review by scpeters — APPROVED** [review]

**Pull request merged**